fe/fe-core/src/main/java/org/apache/doris/datasource/EsExternalDataSource.java

```diff
@@ -166,7 +167,10 @@ public List<String> listDatabaseNames(SessionContext ctx) {
 
     @Override
     public List<String> listTableNames(SessionContext ctx, String dbName) {
-        return esRestClient.getIndexes();
+        List<String> indexes = esRestClient.getIndexes().stream().distinct().collect(Collectors.toList());
+        esRestClient.getAliases().entrySet().stream().filter(e -> indexes.contains(e.getKey()))
```

Comment:
Do we treat `alias` as an independent index?
Should we keep the correspondence between the alias and the real index？

Reply:
Currently, the method of geting the mapping is compatible. Alias and index can be queried. Therefore, the mapping relationship is not maintained.